src/load.ts

```diff
@@ -0,0 +1,75 @@
+import has from './has';
+import Promise from './Promise';
+
+declare var require: Function;
+declare var define: any;
```

Comment:
I have noticed we've been adding these two declarations in various places (actually, usually when I see us declare `define`, it includes the `amd` key).  Should we centralize them somewhere?  (Am I wrong to be concerned about clobbering/conflicts?)